src/simulation/thermal/sensorThermal/sensorThermal.cpp

```diff
@@ -119,15 +115,13 @@ bool SensorThermal::readMessages()
 void SensorThermal::UpdateState(uint64_t CurrentSimNanos)
 {
 
-    //! - Only update the thermal status if we were able to read in messages.
-    if(this->readMessages())
-    {
-        this->evaluateThermalModel(CurrentSimNanos*NANO2SEC);
-    } else {
-        /* if the read was not successful then zero the output message */
-        this->temperatureMsgBuffer = this->temperatureOutMsg.zeroMsgPayload;
-    }
+    //! - Read in messages
+    this->readMessages();
+    
+    //! - Evaluate model
+    this->evaluateThermalModel(CurrentSimNanos*NANO2SEC);
 
+    //! - Write output
     this->writeMessages(CurrentSimNanos);
 
     return;
```

Comment:
please remove this `return` statement as well.  We have started to clean up BSK code as we come across of this.

Reply:
Done.